### PR TITLE
OpenShift compatibility

### PR DESCRIPTION
**Pull request by iivorait**
Improved compatibility with OpenShift

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -63,12 +62,20 @@ COPY . /app
 RUN yarn build
 
 # =============================
-FROM nginx:1.17 as production
+FROM rhel8/nginx-116 as production
```
Comment (vikoivun) [inline Dockerfile]: This is problematic, as it becomes impossible to build the image outside OpenShift. Would be it okay to use explicit URL here? Ie:
registry.redhat.io/rhel8/nginx-116
That would allow building elsewhere as well.

File: .prod/nginx.conf [hunk 2]
```diff
@@ -1,22 +1,39 @@
-server {
-    listen       80;
-    server_name  localhost;
+pid        /tmp/nginx.pid; #for running as non-root 
 
-    location /healthz {
-        return 200 'OK';
-    }
+events {
+    worker_connections  4096;  ## Default: 1024
```
Comment (vikoivun) [inline .prod/nginx.conf:4]: What is the purpose of this change? Looks benign, but would be nice to know why.

**Review by vikoivun — CHANGES_REQUESTED** [review]

File: Dockerfile (hunk 1 above)
Comment (iivorait) [reply Dockerfile]: Changed this

File: .prod/nginx.conf (hunk 2 above)
Comment (iivorait) [reply .prod/nginx.conf:4]: Probably came from a config template for high traffic websites: https://stackoverflow.com/questions/56723449/nginx-error-1024-worker-connections-are-not-enough

Comment (Sukriva) [reply .prod/nginx.conf:4]: Does this need to be changed or can it be left as is?

Comment (tituomin) [reply .prod/nginx.conf:4]: Can be left as is.

**Review by tituomin — APPROVED** [review]

**Review by tituomin — APPROVED** [review]

**Review by vikoivun — APPROVED** [review]
